index.html

```diff
@@ -28,19 +28,22 @@
 
 <body>
     <div class="container">
-        <header class="header-flex align-header">
+        <header class="header-flex align-header static">
             <a href="">Tochukwu</a>
-            <div class="menubar">
-                <div class="vector1"></div>
-                <div class="vector2"></div>
-                <div class="vector3"></div>
+            <div class="menubar" id="mobile_menu">
+                <div class="vector"></div>
+                <div class="vector"></div>
+                <div class="vector"></div>
```

Comment:
- [ ] Please link the menu options to the appropriate sections of the page. The portfolio option should take you to the portfolio(My projects) section, but it doesn't, the about option is instead taking you to "My projects" section. Please kindly fix that, as it is part of the project requirements.
![xtra](https://user-images.githubusercontent.com/65852621/136024527-ed391f56-9e4a-4bcc-8d48-f7f310393ab4.png)